target/linux/lantiq/base-files/etc/board.d/01_leds

```diff
@@ -55,6 +55,9 @@ WBMR300)
 	ucidef_set_led_switch "lan3" "LAN3" "wbmr300:green:lan3" "switch0" "0x20"
 	ucidef_set_led_default "router" "router" "wbmr300:green:router" "1"
 	;;
+RS353)
+        ucidef_set_led_default "dsl" "dsl" "$(lantiq_get_dt_led_chosen dsl)" "0"
```

Comment:
That wont work. I've changed the dsl led logic for all lantiq boards some time ago to the following:

In case only one LED exists for internet (pppoe) and dsl line state, use the led to show the pppoe connection status (on if pppoe connection is established, blink on rx/tx via pppoe connection, off if no pppoe connection is established).

In case a LED for internet and a LED for xdsl line status exists, use the internet LED the same way as descibed above, use the xDSL LED for the xDSL line state (blink on handshake, on if line is in sync, off if no signal).

For sure individually, the defaults can be changed for a running system to the values you have pasted but that is and should be the default.

Long story short, drop this line!